packages/salesforcedx-vscode-aura/src/commands/quickpick/quickpick.ts

```diff
@@ -0,0 +1,55 @@
+import { paramCase } from 'change-case';
+import opn = require('opn');
+import { Uri, window, workspace } from 'vscode';
+import { LanguageClient } from 'vscode-languageclient';
+import { LwcNode } from '../../views/lwc-node';
+import { TagItem } from './tag-item';
+
+export function createQuickOpenCommand(client: LanguageClient) {
+    return (arg: any) => {
+        if (arg instanceof LwcNode) {
+            const node = arg as LwcNode;
+            const tag: string = node.label;
+            opn(`https://developer.salesforce.com/docs/component-library/bundle/${tag}`);
+            return;
+        }
+        console.log('Waiting for LSP to be ready');
```

Comment:
If these type of messaging is needed across the extension I'd suggest you add all the text on a `i18n.ts` file. It makes it easier for our doc writers to contribute and for translating the text (in the future). You can see examples of this on each package's `messages` folder.